core/src/com/dragons/game/model/blockFactory/blocks/Block.java

```diff
@@ -2,20 +2,35 @@
 
 import com.badlogic.gdx.graphics.Texture;
 import com.badlogic.gdx.physics.box2d.Body;
+import com.badlogic.gdx.physics.box2d.World;
 import com.dragons.game.model.Subject;
+import com.dragons.game.utilities.IModelType;
 import com.dragons.game.view.modelViews.IModelObserver;
 
 public abstract class Block extends Subject {
 
-    // TODO: correct fields?
-    private float x, y;
-    private Body body;
-    private Texture texture;
+    // TODO: correct fields? TILE INSTEAD OF X, Y, WIDTH, HEIGHT?
+    protected float x, y, width, height;
+    protected Body body;
```

Comment:
Fordi world ikke skal her, skal sannsynligvis ikke body her heller. https://www.youtube.com/watch?v=IDZMRDb1A_M&ab_channel=ForeignGuyMike

Mistenker vi må ha en egen service for å mappe objekter til world, men har ikke sett noe gode eksempler på det. Har du noen kilder på hvordan vi kobler sammen world og spesifikke objektklasser?